pkgs/applications/audio/redux/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, stdenv, fetchurl, libX11, alsa-lib, releasePath ? null }:
+
+# To use the full release version (same as renoise):
+# 1) Sign into https://backstage.renoise.com and download the release version to some stable location.
+# 2) Override the releasePath attribute to point to the location of the newly downloaded bundle.
+
+let
+  urlVersion = lib.replaceStrings [ "." ] [ "_" ];
+
+  version = "1.2.2";
+
+  url = if stdenv.hostPlatform.system == "x86_64-linux"
```

Comment:
I think this should be set inside of `src`, segmentation like this makes it a bit difficult to read.